src/crafting.h

```diff
@@ -69,6 +69,8 @@ struct recipe {
         bool hidden;
     };
     std::vector<bookdata_t> booksets;
+    std::set<std::string> flags;
```

Comment:
the monster type flags as a bitset is only an internal represenation. they are still an array of strings in the monster definition (the json). they were promoted to a bitset for efficiency reasons, iirc.

Please don't respond to my short quote from kevin, got read the actual linked discussion where it is in context.